chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -605,6 +605,15 @@ def miimon=(millisecs)
       self
     end
 
+    def xmit_hash_policy
+      sysfs("bonding/xmit_hash_policy")
```

Comment:
@s-t-e-v-e-n-k are you sure this path always exists?

And did I read this correctly that the path resolves to something like `/sys/class/net/enp0s20f0u2/bonding`? Because if that's the case I don't see a `bonding` directory on my laptop for this NIC.